content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
+needed.
+
+<!-- body -->
+
+
+## Enabling the feature
+
+Storage capacity tracking is an *alpha feature* and only enabled when
+the `CSIStorageCapacity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled. A quick check
+whether a Kubernetes cluster supports the feature is to list
+`CSIStorageCapacity` objects with:
+```shell
+kubectl get csistoragecapacities --all-namespaces
+```
+
+If supported, the response will a list of objects or:
+```
+No resources found
+```
+
+If not supported, this error is printed instead:
+```
+error: the server doesn't have a resource type "csistoragecapacities"
+```
+
+In addition to enabling the feature in the cluster, a [CSI
+driver](/docs/concepts/storage/volumes/#csi) deployment also has to
+support it. Please refer to the driver's documentation for
+details. The feature is not supported for non-CSI storage systems.
+
+Without this support, there will be no information about storage
+capacity available through the driver and the scheduler will schedule
+Pods with volumes provided by the driver without looking for capacity
+information.
+
+## API
+
+There are two API extensions for this feature:
+- [`CSIStorageCapacity` objects](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#csistoragecapacity-v1alpha1-storage-k8s-io):
+  these get produced by a CSI driver in the namespace
+  where the driver is installed. Each object contains capacity
+  information for one storage class and defines which nodes have
+  access to that storage.
+- [The `CSIDriverSpec.StorageCapacity` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#csidriverspec-v1-storage-k8s-io):
+  when set to `true`, the Kubernetes scheduler will consider storage
+  capacity for volumes that use the CSI driver.
+
+## Scheduling
+
+Storage capacity information is used by the Kubernetes scheduler if:
+- the `CSIStorageCapacity` feature gate is true,
+- a Pod uses a volume that has not been created yet,
+- that volume uses a storage class which references a CSI driver and
+  uses [`WaitForFirstConsumer` volume binding
```

Comment:
```suggestion
  uses `WaitForFirstConsumer` [volume binding
```
?

Reply:
Yes, better as the link itself is not specific to `WaitForFirstConsumer` (no anchor for it).